### PR TITLE
[Footer] - add a footer to the dapp2

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.atlassian.net/browse/SOV-827

### CODE REVIEW

File: apps/frontend/src/app/3_organisms/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      href: '#',
+      name: 'Privacy policy',
+    },
+  ];
+
+  return (
+    <UIFooter
+      leftContent={
+        <SovrynLogo
+          dataAttribute="logo"
```
Comment (soulBit) [inline apps/frontend/src/app/3_organisms/Footer/Footer.tsx:49]: Pass in "Powered by Bitcoin" text and small Sovryn logo (without "Sovryn" text) here, from Figma design

File: apps/frontend/src/app/5_pages/App/App.tsx [hunk 2]
```diff
@@ -20,7 +21,7 @@ function App() {
     useWalletConnect();
 
   return (
-    <>
+    <div className="relative">
```
Comment (soulBit) [inline apps/frontend/src/app/5_pages/App/App.tsx]: This breaks the footer sticking to bottom of screen, and positioning of Osano cookie consent button

image

File: apps/frontend/src/app/3_organisms/Footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+export const Footer: FC = () => {
+  const links = [
+    {
+      id: 'start',
+      href: 'https://wiki.sovryn.app/en/getting-started',
+      name: 'Getting started',
+    },
+    {
+      id: 'alfa',
```
Comment (soulBit) [inline apps/frontend/src/app/3_organisms/Footer/Footer.tsx]:
```suggestion
      id: 'dapp-alpha',
```

File: apps/frontend/src/app/3_organisms/Footer/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+import { Link, Footer as UIFooter } from '@sovryn/ui';
+
+import { SocialLinks } from '../../2_molecules';
+import { SovrynLogo } from '../../2_molecules/SovrynLogo/SovrynLogo';
+
+export const Footer: FC = () => {
+  const links = [
+    {
+      id: 'start',
+      href: 'https://wiki.sovryn.app/en/getting-started',
```
Comment (soulBit) [inline apps/frontend/src/app/3_organisms/Footer/Footer.tsx]: Move this link and the others in this file to `apps/frontend/src/utils/constants.ts`

**Review by soulBit — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/2_molecules/SovrynLogo/SovrynLogo.tsx [hunk 5]
```diff
@@ -21,21 +24,24 @@ export const SovrynLogo: React.FC<SovrynLogoProps> = ({
...
   const Logo = useMemo(
     () => (
-      <img
-        src={image}
-        className={className}
-        alt={text}
-        title={text}
-        {...applyDataAttr(dataAttribute)}
-      />
+      <div className={classNames(styles.logo)}>
```
Comment (creed-victor) [inline apps/frontend/src/app/2_molecules/SovrynLogo/SovrynLogo.tsx]:
```suggestion
      <div className={styles.logo}>
```

**Review by creed-victor — CHANGES_REQUESTED** [review]

**Review by soulBit — APPROVED** [review]

**Review by creed-victor — APPROVED** [review]

**Review by LunaSov — APPROVED** [review]

**Pull request merged**